src/plugins/rel/rel.js

```diff
@@ -140,11 +559,28 @@
             step.relative.z = 0;
         }
 
-        // Apply relative position to absolute position, if non-zero
-        // Note that at this point, the relative values contain a number value of pixels.
-        step.x = step.x + step.relative.x;
-        step.y = step.y + step.relative.y;
-        step.z = step.z + step.relative.z;
+        // Once rotate-x/rotate-y/rotate-z is set, all three must be set or use default 0
+        var useRelativeRotate = data.rotateX === undefined &&
+            data.rotateY === undefined &&
+            data.rotateZ === undefined;
```

Comment:
I'm not convinced this restriction is justified. If a user wants to set `rotate-x="90" rotate-rel-y="45"` why should we forbid this?

Reply:
`rel-rotate-*` is based on the status of previous slide, and `rotate-*` is not, the final rotation may be applied to any axis depends on the rotation of previous slide.  I can't image what's the propose and how to mix them.

 If we allow the mixing, maybe we should finish the relative rotation, and then apply global rotation? Maybe it will looks like space distortion, just like we place objects in the plain space, and then distort the space. If we first add the angle and then do the rotation, I just can't image what the result is.

Reply:
As an example, and this is how `rel-x/y/z` work too:

    <div class="step" data-rotate-x="45" data-rotate-y="45" >
    <div class="step" data-rel-rotate-x="10" data-rotate-y="10" >

The effective absolute values for the second step are now `rotate-x=55` and `rotate-y=10`.